roles/foreman_repositories/tasks/redhat.yml

```diff
@@ -18,13 +18,40 @@
   tags:
     - packages
 
+- name: 'Enable foreman:el8 module'
+  command: dnf module enable -y foreman:el8
+  # can't use the `dnf` module for modules without a default stream
+  # https://github.com/ansible/ansible/issues/56504
+  # https://github.com/ansible/ansible/issues/64852
+  args:
+    warn: False
+    creates: /etc/dnf/modules.d/foreman.module
+  when:
+    - foreman_repositories_version == "nightly" or foreman_repositories_version is version('3.2', '>=')
+    - ansible_distribution_major_version == '8'
+  tags:
+    - packages
+
 - name: 'Setup Katello {{ foreman_repositories_katello_version }} Repository'
   yum:
     name: https://yum.theforeman.org/katello/{{ foreman_repositories_katello_version }}/katello/el{{ ansible_distribution_major_version }}/x86_64/katello-repos-latest.rpm
     disable_gpg_check: true
     state: present
   when: foreman_repositories_katello_version is defined
 
+- name: 'Enable katello:el8 module'
+  command: dnf module enable -y katello:el8
+  # can't use the `dnf` module for modules without a default stream
+  # https://github.com/ansible/ansible/issues/56504
+  # https://github.com/ansible/ansible/issues/64852
+  args:
+    warn: False
+    creates: /etc/dnf/modules.d/katello.module
+  when:
+    - foreman_repositories_katello_version is defined
+    - foreman_repositories_katello_version == "nightly" or foreman_repositories_katello_version is version('4.4', '>=')
+    - ansible_distribution_major_version == '8'
+
 - when:
     - foreman_repositories_katello_version is defined
     - ansible_distribution_major_version == '8'
```

Comment:
Shouldn't this be scoped to `< 4.4`? At least, `pki-core` (and possibly powertools`) shouldn't be needed anymore, right?

Reply:
pki-core true, powertools are still needed (thats a repo, not a module)

but I wanted to keep the diff small.

Reply:
implemented now